pages/_app.js

```diff
@@ -26,7 +26,8 @@ function MyApp({ Component, pageProps }) {
       frontendApi={process.env.NEXT_PUBLIC_CLERK_FRONTEND_API}
       scriptUrl={process.env.NEXT_PUBLIC_CLERK_JS}
       autoCredential
```

Comment:
Remove autoCredential?  We haven't documented this because we haven't found a way for it to play nice with multi session mode and the react helpers for queries.

Reply:
Agreed.